anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/detect/breakout/algo/edmx/EdmxDetector.java

```diff
@@ -32,6 +32,7 @@
 
 @Slf4j
 public final class EdmxDetector implements BreakoutDetector {
+    private final String NAME = "edmx";
```

Comment:
If we do it this way (and fine by me), wouldn't we want this to be `static` as well?